stepconf/stepconf.go

```diff
@@ -160,6 +172,15 @@ func setField(field reflect.Value, value, constraint string) error {
 		field.SetFloat(f)
 	case reflect.Slice:
 		field.Set(reflect.ValueOf(strings.Split(value, "|")))
+	case reflect.Ptr:
```

Comment:
Could move this check before the switch case and (or a new function) to avoid a recursive function call.